chef/data_bags/crowbar/template-provisioner.json

```diff
@@ -215,14 +215,15 @@
           "debug": "debug"
         }
       },
+      "chef_splay": 20,
```

Comment:
Can we bump the default to 300? 20s is really to slow.

Reply:
We agreed with @matelakat that defaults change can be done as separate PR. Let me know if this change is needed here.

Reply:
That should be a separate PR